### DISCUSSION TITLE
Use SSH keys to install private repositories

### DISCUSSION THREAD

**Discussion by Fyrlex** [community/community#43789] [Actions] [answered]
### Select Topic Area

Question

### Body

Hi! Just started learning how to use Actions and came across an issue with private repositories in my organization. Here's the setup:

Private repository `luyx-management-discord` has a workflow to deploy the source code to a self-hosted runner. The repository is trying to also install another private repository `loggage` in the organization. My goal is to use SSH keys rather than account tokens to install this package (not potentially exposing a token in the package.json). I've tried setting up SSH keys and deploy keys but the workflow job is showing that it still cannot install `loggage` because it cannot access it.

![image](https://user-images.githubusercontent.com/31144090/211435503-9baff002-9706-43ee-a396-2689d7beee15.png)

Tried using this setup and adding the key to the repo. No success.
```
Host github.com
        Hostname github.com
        IdentityFile=/home/user/.ssh/repo-0_deploy_key
```

I'm assuming my setup just isn't complete and I couldn't find help for this specific situation.

Appreciate the help, let me know if more information is needed.

**Comment by Fyrlex** [comment, marked as answer]
solved

**Reply by sanskar-19** [reply]
How did you solve the same?
